frontend/src/App.js

```diff
@@ -1,23 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import NaviBar from './components/Navibar';
+import {BrowserRouter, Route, Switch} from 'react-router-dom';
+import Login from './pages/Login';
+import Register from './pages/Register';
+import Home from './pages/Home';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <BrowserRouter>
+        <NaviBar />
+        <Container className="my-3">
+					
+							<Switch>
+								<Route exact path="/" component={Home} />
+								<Route exact path="/register" component={Register} />
+								<Route exact path="/login" component={Login} />
+					
+							</Switch>
+
+						</Container>
+
```

Comment:
Proper spacing lang dinhi, dapat 2 space indentation ra na bast sa javascript. You can use prettier extension sa vscode sa para ma fix.

expected outpue:
```suggestion
        <Container className="my-3">
          <Switch>
            <Route exact path="/" component={Home} />
            <Route exact path="/register" component={Register} />
            <Route exact path="/login" component={Login} />
          </Switch>
        </Container>
```